src/commands/Utility/quote.ts

```diff
@@ -142,6 +149,34 @@ async function addQuote(interaction: Discord.CommandInteraction, collection: Col
 		author = interaction.options.getUser("author", true),
 		url = interaction.options.getString("url", false)
 
+	let pictureUrl = interaction.options.getString("pictureUrl", false) ?? undefined
+
+	const urlSplit = url?.split("/")
+
+
+	if (urlSplit) {
+		if (urlSplit.length === 7) {
+			const message = await (client.channels.cache.get(urlSplit[5]) as Discord.TextChannel | undefined)?.messages.fetch(urlSplit[6])
```

Comment:
This needs to be catched. Should be a .then.catch instead

Reply:
Fair enough, fixed